registry/root.go

```diff
@@ -18,6 +18,7 @@ func init() {
 	RootCmd.AddCommand(ServeCmd)
 	RootCmd.AddCommand(GCCmd)
 	GCCmd.Flags().BoolVarP(&dryRun, "dry-run", "d", false, "do everything except remove the blobs")
+	GCCmd.Flags().BoolVarP(&manifestswithouttags, "delete-manifests", "m", false, "delete manifests which does not have tag attached")
```

Comment:
`delete-untagged`. The manifest still exist. What we are trying to delete is the manifests that are not currently referenced via tag.